data/projects.yml

```diff
@@ -157,22 +157,22 @@ categories:
         url: https://github.com/Esri/arcgis-runtime-toolkit-qt
         displayLang: QML
         searchLang: qml
-        stars: 105
+        stars: 124
       - title: ArcGIS Toolkit .NET
         description: Controls and utilities you can use with the ArcGIS Runtime SDK for .NET.
         url: //github.com/Esri/arcgis-toolkit-dotnet
         displayLang: C#
         searchLang: csharp
-        stars: 187
-      - title: Dynamic Situational Awareness
-        description: Built using the ArcGIS Runtime SDK for Qt, DSA highlights developer best practices for the specific workflows for in-vehicle (mounted) and field (handheld/dismounted) teams who need to dynamically understand their environment and the situation around them.
-        url: //github.com/Esri/dynamic-situational-awareness-qt
-        displayLang: C++
-        searchLang: c%2B%2B
-        stars: 49
-      - title: Nearby-Places Android
-        description: Locate nearby places with Android devices. Built with the ArcGIS Runtime SDK for Android. With some minor customization, you can change the types of places displayed.
-        url: //github.com/Esri/nearby-android
+        stars: 209
+      - title: ArcGIS Runtime Toolkit for iOS
+        description: The ArcGIS Runtime SDK for iOS Toolkit contains components that will simplify your iOS app development.
+        url: //github.com/Esri/arcgis-runtime-toolkit-ios
+        displayLang: swift
+        searchLang: swift
+        stars: 50
```

Comment:
```suggestion
      - title: ArcGIS Maps SDK for Swift toolkit
        description: The ArcGIS Maps SDK for Swift toolkit contains components that will simplify your iOS app development.
        url: //github.com/Esri/arcgis-maps-sdk-swift-toolkit
        displayLang: swift
        searchLang: swift
        stars: 25
```

Reply:
Runtime iOS is in mature support, we are asking users to migrate to the Maps SDK for Swift